Mu2eG4/geom/geom_2021_PhaseI_v02.txt

```diff
@@ -9,7 +9,6 @@ bool hasProtonAbsorber  = true;
 bool hasTSdA           = true;
 bool hasExternalShielding = true;
 bool hasDiskCalorimeter = true;
-bool hasBFieldManager   = true;
```

Comment:
One minor consideration: Is it better to keep this parameter and set it to false explicitly? I see that it defaults to false in the code so it doesn't really matter